rfcs/019-platform_reliability/reliability.md

```diff
@@ -0,0 +1,29 @@
+# Reliability
+
+In order that we can be confident that what we deploy will behave as we expect we should look at:
+
+## Pipeline reliability
+
+In order to verify that pipeline deployments will behave and are behaving as expected we should:
+
+- Test pipeline changes that require re-indexes before new indexes are promoted into production view.
+
+- Test the pipeline is performing as expected when updates flow through it both:
+  - In production
+  - In integration tests
+  
```

Comment:
This feels like a reactionary bit of documentation that doesn't have any action bar "make sure you do it properly". 

But the integration tests between services that make up a stack feel important.